core/src/main/java/com/graphhopper/routing/util/CarFlagEncoder.java

```diff
@@ -158,6 +158,9 @@ public int defineWayBits(int index, int shift) {
 
     protected double getSpeed(ReaderWay way) {
         String highwayValue = way.getTag("highway");
+        if (way.hasTag("motorroad", "yes")) {
```

Comment:
Or maybe should use the motorroad check if highway is empty?

@miklcct do you have a real world case for this? Or where did you stumble over a problem?